docs/source/how-to-discussions-and-pull-requests.mdx

```diff
@@ -0,0 +1,129 @@
+# Interact with Discussions and Pull Requests 
+
+The `huggingface_hub` library provides a Python interface to interact with Pull Requests and Discussions on the Hub.
+Visit [the dedicated documentation page](https://huggingface.co/docs/hub/repositories-pull-requests-discussions)
+for a deeper view of what Discussions and Pull Requests on the Hub are, and how they work under the hood.
+
+## Retrieve Discussions and Pull Requests from the Hub
+
+The `HfApi` class allows you to retrieve Discussions and Pull Requests on a given repo:
+
+```python
+>>> from huggingface_hub import get_repo_discussions
+>>> for discussion in get_repo_discussions(repo_id="bigscience/bloom-1b3"):
+...     print(f"{discussion.num} - {discussion.title}, pr: {discussion.is_pull_request}")
```

Comment:
Out of curiosity, is this an alias for the `HfApi.get_repo_discussions` method ?
I found it confusing to say that we are using the `HfApi` class and then in the example it is not used (at least not explicitly). I get it though if it's just to keep the example more concise and still functional.

Reply:
> is this an alias for the HfApi.get_repo_discussions method

That is correct